test/functional/fixtures/click-test.js

```diff
@@ -0,0 +1,33 @@
+var runTest = require('../test-runner.js').runTests;
+var expect  = require('chai').expect;
+
+
+describe('api click test', function () {
+    it('Should fail when the first argument is invisible', function () {
+        var testSuccessed = false;
+
+        return runTest('./test/functional/fixtures/click.test.js', '[SHOULD FAIL] when the first argument is invisible')
+            .then(function () {
+                testSuccessed = true;
+
+                throw new Error();
+            })
+            .catch(function (errs) {
+                if (testSuccessed)
```

Comment:
this looks not good. I think we should use `done(err)` way instead of return the promise in this case